### PR TITLE
remove remaining doTests functions

### PR DESCRIPTION
**Pull request by winsvega**
--filltests --fillchain
works simultaneously. creating both stateTest and Blockchain test

WIP:  have to fix BCGeneralStateTests

### CODE REVIEW

File: test/tools/jsontests/StateTests.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+*/
+/** @file StateTests.h
+ * StateTests functions.
+ */
+
+#pragma once
+#include <test/tools/libtesteth/TestSuite.h>
+
+
+using namespace dev;
```
Comment (pirapira) [inline test/tools/jsontests/StateTests.h]: Avoid `using namespace` in the header files.

File: test/tools/libtesteth/ImportTest.cpp [hunk 2]
```diff
@@ -181,12 +182,11 @@ bytes ImportTest::executeTest()
...
-			fs::path const tmpFillerName = getTestPath() / fs::path("src/GenStateTestAsBcTemp") / fs::path(TestOutputHelper::caseName()) / fs::path(testname + "Filler.json");
-			writeFile(tmpFillerName, asBytes(json_spirit::write_string((json_spirit::mValue)json, true)));
-			dev::test::executeTests(testname, "/BlockchainTests/GeneralStateTests/" + TestOutputHelper::caseName(),
-											 "/GenStateTestAsBcTemp/" + TestOutputHelper::caseName(), dev::test::doBlockchainTestNoLog);
-
+			//Write a blockchain test .json file from dynamic filler 'json' that is produced from state test filler
+			BCGeneralStateTestsSuite genSuite;
+			fs::path const testFile = genSuite.getFullPath(TestOutputHelper::caseName()) / fs::path(testname + ".json");
+			json_spirit::mValue const generatedJson = genSuite.doTests((json_spirit::mValue)json, true);
+			writeFile(testFile, asBytes(json_spirit::write_string(generatedJson, true)));
```
Comment (pirapira) [inline test/tools/libtesteth/ImportTest.cpp]: Why is `executeTests` not used anymore?  What happened to `GenStateTestAsBcTemp`?

File: test/tools/libtesteth/TestSuite.h [hunk 3]
```diff
@@ -47,6 +49,12 @@ class TestSuite
 	// Copy .json tests from the src folder to the dest folder because such test is crafted manually and could not be filled.
 	// Used in bcForgedTest, ttWrongRLP tests and such.
 	void copyAllTestsFromFolder(std::string const& _testFolder) const;
+
+	// Return full path to folder for tests from _testFolder
+	fs::path getFullPathFiller(std::string const& _testFolder) const;
+
+	// Structure  <suiteFolder>/<testFolder>/<test>.json
+	fs::path getFullPath(std::string const& _testFolder) const;
```
Comment (pirapira) [inline test/tools/libtesteth/TestSuite.h]: These can stay `private` I guess.

Comment (winsvega) [reply test/tools/libtesteth/TestSuite.h]: it is used to save the hive test file 
executeTest is gona be removed.

Comment (pirapira) [reply test/tools/libtesteth/TestSuite.h]: OK.

File: test/tools/libtesteth/TestSuite.h [hunk 4]
```diff
@@ -19,7 +19,9 @@
  */
 
 #pragma once
+#include <boost/filesystem/path.hpp>
 #include <test/tools/libtesteth/JsonSpiritHeaders.h>
+namespace fs = boost::filesystem;
```
Comment (gumb0) [inline test/tools/libtesteth/TestSuite.h]: Avoid aliases in global namespace in the header files. You can put it inside `namespace dev { namespace test {`

**Pull request merged**